css/main.css

```diff
@@ -0,0 +1,82 @@
+.logo {
+  background: linear-gradient(90deg, #00FFFF 0%, #AA96CA 46.35%, #FF6B98 100%);
+  width: 75px;
+  height: 75px;
+  border-radius: 50px;
+}
+
+.nav-link {
+  color: #B6A1C4;
+}
+
+.nav-link:hover, .nav-link:active {
+  color: #B6A1C4;
+  filter: brightness(1.2);
+}
+
+.heading {
+  color: #FF6B98;
+}
+
+.email-form {
+  background: linear-gradient(90deg, #00FFFF 0%, #AA96CA 46.35%, #FF6B98 100%);
+  padding: 2px;
+}
+
+.btn {
+  background-color: transparent;
+  border: none;
+  color: white;
+}
+
+.social {
+  padding-top: 10px;
+}
+
+.social-link {
+  display: block;
+  background: #B6A1C4;
+  width: 35px;
+  height: 35px;
+  border-radius: 50px;
+}
+
+.main {
+  margin-top: 50px;
+  padding-left: 200px;
+  padding-right: 200px;
+}
+
+.past-link {
+  display: block;
+  position: relative;
+  margin-bottom: 30px;
+}
+
+.past-link img {
+  height: 300px;
+  width: 100%;
+}
+
+.past-link:hover img {
+  filter: brightness(.8);
+  transition: .3s ease;
+}
+
+
```

Comment:
This is quite nitpicky, but can we keep the spacing between CSS declarations consistent? So 1 empty line between. We'll add a linter to do this automatically later 🙂